libraries/AP_Notify/Buzzer.cpp

```diff
@@ -196,12 +101,33 @@ void Buzzer::update()
     // if vehicle lost was enabled, starting beep
     if (AP_Notify::flags.vehicle_lost) {
```

Comment:
If vehicle is lost this will be constantly restarting. Same with battery failsafe below.

Reply:
Fixed.